src/sentry/snuba/entity_subscription.py

```diff
@@ -206,6 +228,18 @@ def build_snuba_filter(
     def get_entity_extra_params(self) -> Mapping[str, Any]:
         return {"organization": self.org_id}
 
+    def aggregate_query_results(
+        self, data: List[Dict[str, Any]], alias: Optional[str] = None
+    ) -> List[Dict[str, Any]]:
+        col_name = alias if alias else CRASH_RATE_ALERT_AGGREGATE_ALIAS
+        if data[0][col_name] is not None:
```

Comment:
Might be unnecessary because by definition all subscriptions (except metrics) always return one row because its an aggregate but I can add the assert as a check